app/assets/images/payment_icons/payflex.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24"><defs><style>.cls-3{fill:#5592cd}</style></defs><path d="M22 9a15.15 15.15 0 000 6 15.08 15.08 0 017.71 4.14 10.12 10.12 0 010-14.28A15.08 15.08 0 0122 9m-6 0a14.39 14.39 0 003 .29A15.11 15.11 0 0022 9a15.08 15.08 0 014.14-7.71 10.05 10.05 0 01-7.14 3 10.07 10.07 0 01-7.14-2.95A15.08 15.08 0 0116 9" fill="#81d3f3"/><path d="M22 15a15.9 15.9 0 00-3-.3 15.11 15.11 0 00-3 .3 15.08 15.08 0 01-4.14 7.71 10.12 10.12 0 0114.28 0A15.08 15.08 0 0122 15m-6 0a15.15 15.15 0 000-6 15.08 15.08 0 01-7.71-4.14 10.12 10.12 0 010 14.28A15.08 15.08 0 0116 15" fill="#c5b1d4"/><path class="cls-3" d="M8.29 1.29a2.52 2.52 0 000 3.57A15.08 15.08 0 0016 9a15.08 15.08 0 00-4.14-7.71 2.52 2.52 0 00-3.57 0"/><path d="M8.29 22.71a2.52 2.52 0 003.57 0A15.08 15.08 0 0016 15a15.08 15.08 0 00-7.71 4.14 2.52 2.52 0 000 3.57" fill="#8c64aa"/><path class="cls-3" d="M29.71 22.71a2.52 2.52 0 01-3.57 0A15.08 15.08 0 0122 15a15.08 15.08 0 017.71 4.14 2.52 2.52 0 010 3.57"/><path d="M29.71 1.29a2.52 2.52 0 010 3.57A15.08 15.08 0 0122 9a15.08 15.08 0 014.14-7.71 2.52 2.52 0 013.57 0" fill="#29aae1"/></svg>
```

Comment:
Please read the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:

- We are missing the `width`, `height` and `viewBox`
- Please remove any unnecessary `id` attributes
- Please `<style>` tags in favor of attributes.

Also, is there a branding guideline available?

Reply:
I need to make svg icon as per the guideline and generate Pull request again right ?

Reply:
👋 Hey @hemabadgali, you should be able to update this podcast directly. That said, yes, will likely need to generate an SVG to works according to the guidelines (has `width`, `height`, and `viewBox`, no extra `id`s, etc.) @tauthomas01 specified.

Reply:
I have committed latest Payflex logo.

Reply:
with changes as suggested

Reply:
> The 'payme' SVG file does not have the appropriate <title> value

Looks like some [tests failing in CI](https://github.com/activemerchant/payment_icons/pull/414/checks?check_run_id=1909772014). Seemingly need to fix the icon title.

Reply:
Pls refer payflex.svg
I committed that, not payme.svg

Reply:
My apologies, I think the unrelated test failed because this PR/branch is out of date. Can you rebase it on latest master?

As for test failures for this PR, I'm seeing the following.

```
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/Users/larouxn/src/github.com/activemerchant/payment_icons/test/unit/payment_icon_test.rb:68]:
{:message=>"The 'payflex' SVG file should have a single <title> tag"}.

PaymentIconTest#test_Every_payment_icon_SVG_is_38x24_pixels [/Users/larouxn/src/github.com/activemerchant/payment_icons/test/unit/payment_icon_test.rb:46]:
{:message=>"The 'payflex' SVG file must be 38 pixels wide"}.
```
<img width="1920" alt="Screen Shot 2021-02-16 at 23 59 00" src="https://user-images.githubusercontent.com/1557529/108080440-35dc0280-70b3-11eb-85f0-56e3b938eb84.png">

Test failures should be viewable in these CI actions towards the bottom of the PR.
<img width="929" alt="Screen Shot 2021-02-17 at 00 03 27" src="https://user-images.githubusercontent.com/1557529/108080959-c4508400-70b3-11eb-934b-8f136d84b46d.png">

Reply:
@larouxn when I rebase get following message
"Current branch payflexicon is up to date."

Can we close this ASAP?

Reply:
Since long its open

Reply:
I'm seeing `hemabadgali:payflexicon` branch is 4 commits behind `activemerchant:master`.
<img width="531" alt="Screen Shot 2021-02-17 at 14 18 01" src="https://user-images.githubusercontent.com/1557529/108159829-2ef0d680-712b-11eb-87e7-0ee951dc4d44.png">

Also, there are failing tests that will need be addressed such as we need to add a title to the SVG.
> PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/Users/larouxn/src/github.com/activemerchant/payment_icons/test/unit/payment_icon_test.rb:68]:
{:message=>"The 'payflex' SVG file should have a single <title> tag"}.
<img width="875" alt="Screen Shot 2021-02-17 at 14 18 25" src="https://user-images.githubusercontent.com/1557529/108159869-3fa14c80-712b-11eb-987c-51ead955eef6.png">

Reply:
Sure will Add title and Commit again

Do I need to rebase forked branch original master branch

Reply:
@larouxn I have rebase and push latest payflex.svg with title.
Can you pls check from your end?

Reply:
Hmm, I think you need to update your fork's master branch as this branch is still 4 commits behind.
<img width="517" alt="Screen Shot 2021-02-17 at 16 21 22" src="https://user-images.githubusercontent.com/1557529/108169630-52704d00-713c-11eb-8f26-7da2d88ea637.png">

Can likely do according to this guide. 🙇‍♂️ 
https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a

Reply:
But I have already followed all the steps

Reply:
I followed all the steps mentioned in link
Can you pls check one more time?

Reply:
Looking at https://github.com/hemabadgali/payment_icons/tree/payflexicon, it seems it's still behind by 4 commits.

Would it be alright if I created a PR to add Payflex? I can reference this PR and credit you in the description.

Reply:
hay @larouxn Yes please go ahead

Reply:
Alright, opened https://github.com/activemerchant/payment_icons/pull/418 to add Payflex. Please let me know if that PR looks good to you.

Reply:
yes fine

Reply:
All I want to upload Payflex Payment Gateway Icon for to display in shopify website footer

Reply:
Understood. Will be possible once merged, released, and updated across Shopify.

Reply:
How much time it will take?

Reply:
anything Pending from my end ?

Reply:
Committed logo with all changes mentioned

Reply:
We need to fix the icon style first. I will try to get it fixed and live by the end of next week. 
https://github.com/activemerchant/payment_icons/pull/418#pullrequestreview-592234746

Reply:
@larouxn latest Icon I committed has all changes mentioned in #418 (review).

Reply:
Thanks @hemabadgali! I've updated #418. Should be able to merge this week.

Reply:
Thanks so much @larouxn 
when it will available on live? How I can display that at shopify footer

Reply:
I'm hoping to have the icon live this week. Footer wise, I believe that's based on your shop's payment methods. Can also be customized. I would reach out to support for guidance.